pyspark_ai/pyspark_ai.py

```diff
@@ -76,7 +76,7 @@ def __init__(
         """
         self._spark = spark_session or SparkSession.builder.getOrCreate()
         if llm is None:
-            llm = ChatOpenAI(model_name="gpt-4", temperature=0)
+            llm = ChatLiteLLM(model_name="gpt-4", temperature=0)
```

Comment:
The default value of the `max_tokens` parameter for ChatLiteLLM is 256. It should be increased to avoid truncation of replies.
eg: `llm = ChatLiteLLM(model_name="gpt-4", temperature=0, max_tokens=2048)`